pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -254,3 +269,31 @@ func (m *managerImpl) synchronize(diskInfoProvider DiskInfoProvider, podFunc Act
 	}
 	glog.Infof("eviction manager: unable to evict any pods from the node")
 }
+
+// reclaimNodeLevelResources attempts to reclaim node level resources.  returns true if thresholds were satisfied and no pod eviction is required.
+func (m *managerImpl) reclaimNodeLevelResources(resourceToReclaim api.ResourceName, observations signalObservations) bool {
```

Comment:
The existing reclaimResources() seems to collide with this method in terms of naming. How about naming the existing reclaimResources() to something like getStarvedResources()?